commands.py

```diff
@@ -190,7 +189,6 @@ async def parseReaction(reaction, inst):
 		# Change Player Role
 		await inst.changePlayerRole(reaction)
   
-  
 	async def parse(message, inst):
 		client = inst.client
 		if message.author == client.user or not message.content.startswith(commands.COMMAND_SYMBOL):
```

Comment:
it looks like you haven't changed the tabs to spaces in commands.py or serverinstance.py